README.md

```diff
@@ -171,20 +173,44 @@ const requestRefresh = async (refresh) => {
 
 // 3. Apply interceptor
 applyAuthTokenInterceptor(axiosInstance, { requestRefresh });  // Notice that this uses the axiosInstance instance.
+```
+### Login/logout
+
+```javascript
+//login.js
 
-// 4. Logging in
+import {
+  isLoggedIn,
+  setAuthTokens,
+  clearAuthTokens,
+  getAccessToken,
+  getRefreshToken,
+} from 'react-native-axios-jwt';
+import { axiosInstance } from '../api';
+
+// 4. Post email and password and get tokens in return. Call setAuthTokens with the result.
 const login = async (params) => {
   const response = await axiosInstance.post('/auth/login', params)
 
   // save tokens to storage
-  await setAuthTokens({
+    await setAuthTokens({
     accessToken: response.data.access_token,
     refreshToken: response.data.refresh_token
   })
 }
 
-// 5. Logging out
-const logout = async () => await clearAuthTokens()
+// 5. Clear the auth tokens from AsyncStorage
```

Comment:
I just made a couple of tiny changes to the readme myself to clarify the code examples a bit more.

```suggestion
// 5. Log out by clearing the auth tokens from AsyncStorage
```